Alignment/OfflineValidation/plugins/OverlapValidation.cc

```diff
@@ -0,0 +1,876 @@
+// -*- C++ -*-
+//
+// Package:    Alignment/OfflineValidation
+// Class:      OverlapValidation
+// 
+/**\ Alignment/OfflineValidation/src/OverlapValidation.cc
+
+ Description: <one line class summary>
+	
+*/
+//
+// Original Authors:  Wolfgang Adam, Keith Ulmer
+//         Created:  Thu Oct 11 14:53:32 CEST 2007
+// $Id: OverlapValidation.cc,v 1.16 2009/11/04 19:40:27 kaulmer Exp $
+//
+//
+ 
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "MagneticField/Engine/interface/MagneticField.h"
+#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+
+#include "CommonTools/Statistics/interface/ChiSquaredProbability.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "CalibTracker/SiStripCommon/interface/SiStripDetInfoFileReader.h"
+
+#include "Geometry/TrackerGeometryBuilder/interface/PixelGeomDetUnit.h"
+#include "Geometry/TrackerGeometryBuilder/interface/PixelGeomDetType.h"
+#include "Geometry/CommonDetUnit/interface/GeomDetType.h"
+#include "Geometry/CommonDetUnit/interface/GeomDet.h"
+#include "Geometry/TrackerGeometryBuilder/interface/RectangularPixelTopology.h"
+
+#include  "DataFormats/TrackReco/interface/TrackFwd.h"
+#include  "DataFormats/TrackReco/interface/Track.h"
+#include  "DataFormats/TrackReco/interface/TrackExtra.h"
+#include  "DataFormats/TrackReco/interface/TrackExtraFwd.h"
+#include "DataFormats/TrackingRecHit/interface/TrackingRecHit.h"
+#include "DataFormats/SiStripDetId/interface/StripSubdetector.h"
+#include "DataFormats/DetId/interface/DetId.h" 
+//#include "DataFormats/SiStripDetId/interface/TIBDetId.h"
```

Comment:
as these DataFormats do not even exist anymore in cmssw, you can remove the commented lines.

Reply:
Done